web-app/src/hoc/Layout/container.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import {
+  faUser,
+  faTachometerAlt,
+  faUnlockAlt,
+} from '@fortawesome/fontawesome-free-solid';
+
+export default Wrapped =>
+  class extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+        menuItems: [
```

Comment:
Why is this being stored in state?

Reply:
I brought this up before with @paul-matchett in a previous PR. I suggested moving it into a nav config. His reasoning was because thinks it would be useful in the future if the user could modify their sidebar to suit their needs. (Pin pages, and whatnot)

I said it was grand because of the reasoning but looking back now I think it's maybe a bit premature to have it in the state.

Reply:
Yeah, I can't see it needing to be modified anytime soon, think it's best stored elsewhere 👍

Reply:
@bgribz I'll fix this in another branch as it's not the focus of this PR.